examples/community/index.test.ts

```diff
@@ -0,0 +1,581 @@
+          35, 52, 51, 54, 48, 68, 70, 34, 36, 56, 102, 50, 99, 99, 53, 100, 55,
+          45, 101, 99, 98, 54, 45, 52, 50, 99, 100, 45, 97, 102, 49, 55, 45, 49,
+          101, 55, 101, 100, 57, 57, 51, 98, 100, 55, 97, 40, 2, 50, 76, 10, 36,
+          50, 48, 50, 101, 50, 53, 102, 50, 45, 52, 56, 97, 99, 45, 52, 101, 50,
+          100, 45, 56, 102, 100, 48, 45, 97, 97, 53, 57, 55, 49, 51, 56, 51, 49,
+          51, 52, 18, 36, 18, 2, 24, 1, 26, 28, 34, 4, 116, 101, 115, 116, 42,
+          4, 116, 101, 115, 116, 50, 7, 35, 50, 54, 65, 54, 57, 65, 58, 5, 240,
+          159, 152, 131, 32, 40, 2, 66, 93, 10, 36, 101, 48, 57, 100, 53, 53,
+          54, 56, 45, 101, 49, 53, 56, 45, 52, 99, 51, 50, 45, 57, 98, 100, 102,
+          45, 52, 100, 51, 54, 57, 98, 101, 97, 48, 48, 55, 57, 18, 53, 10, 36,
+          101, 48, 57, 100, 53, 53, 54, 56, 45, 101, 49, 53, 56, 45, 52, 99, 51,
+          50, 45, 57, 98, 100, 102, 45, 52, 100, 51, 54, 57, 98, 101, 97, 48,
+          48, 55, 57, 18, 13, 116, 101, 115, 116, 45, 99, 97, 116, 101, 103,
+          111, 114, 121, 66, 97, 10, 36, 56, 54, 98, 50, 57, 97, 50, 51, 45, 57,
+          50, 57, 97, 45, 52, 52, 99, 50, 45, 57, 56, 102, 53, 45, 101, 51, 55,
+          48, 51, 97, 48, 48, 52, 100, 57, 50, 18, 57, 10, 36, 56, 54, 98, 50,
+          57, 97, 50, 51, 45, 57, 50, 57, 97, 45, 52, 52, 99, 50, 45, 57, 56,
+          102, 53, 45, 101, 51, 55, 48, 51, 97, 48, 48, 52, 100, 57, 50, 18, 15,
+          116, 101, 115, 116, 45, 99, 97, 116, 101, 103, 111, 114, 121, 45, 51,
+          24, 1, 66, 97, 10, 36, 56, 102, 50, 99, 99, 53, 100, 55, 45, 101, 99,
+          98, 54, 45, 52, 50, 99, 100, 45, 97, 102, 49, 55, 45, 49, 101, 55,
+          101, 100, 57, 57, 51, 98, 100, 55, 97, 18, 57, 10, 36, 56, 102, 50,
+          99, 99, 53, 100, 55, 45, 101, 99, 98, 54, 45, 52, 50, 99, 100, 45, 97,
+          102, 49, 55, 45, 49, 101, 55, 101, 100, 57, 57, 51, 98, 100, 55, 97,
+          18, 15, 116, 101, 115, 116, 45, 99, 97, 116, 101, 103, 111, 114, 121,
+          45, 50, 24, 2, 82, 0, 24, 25,
+        ]),
+        signaturePublicKey: new Uint8Array([
+          4, 172, 65, 157, 172, 154, 139, 187, 88, 130, 90, 60, 222, 96, 238,
+          240, 238, 113, 184, 207, 108, 99, 223, 97, 30, 238, 252, 142, 122,
+          172, 124, 121, 181, 89, 84, 182, 121, 210, 76, 245, 236, 130, 218,
+          126, 217, 33, 202, 242, 64, 98, 138, 155, 251, 52, 80, 197, 17, 26,
+          156, 255, 229, 78, 99, 24, 17,
+        ]),
+        timestamp: new Date(1657640185127),
+      },
+    ])
+  })
+}, 10 * 60 * 1000)
+
+afterEach(async () => {
+  await page.close()
+})
+
+test(
+  'receiving message',
+  async () => {
+    await page.goto('http://localhost:3001')
+
+    // Click text=t#test-messages
+    await page
+      .locator('text=t#test-messages')
+      .click({ timeout: 15 * 60 * 1000 })
+    await page.waitForURL(
+      'http://localhost:3001/30804ea7-bd66-4d5d-91eb-b2dcfe2515b3'
+    )
+
+    await page.evaluate(() => {
```

Comment:
Note: in case of two page sharing global js-waku mock, this fixture creating and "sending"/evaluating would not be necessary – messages would be created and send via selectors and clicking.